src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkAddDetails.tsx

```diff
@@ -12,6 +13,7 @@ import { ScrollView } from "react-native"
 
 export const SubmitArtworkAddDetails = () => {
   const { handleChange, setFieldValue, values } = useFormikContext<ArtworkDetailsFormModel>()
+  const { currentStep } = SubmitArtworkFormStore.useStoreState((state) => state)
```

Comment:
Let's not access the store directly in the step and instead expose `currentStep` instead from `useSubmissionContext`.

```suggestion
  const { currentStep } = useSubmissionContext()
```

Reply:
nice to use this - was thinking about exporting it as well